### DISCUSSION TITLE
Instalation error

### DISCUSSION THREAD

**Discussion by aniltoncabral** [HomeSmartMesh/raspi#43] [Q&A]
after many retries to install raspi, i got the same

FAILED: src/web/CMakeFiles/otbr-web.dir/web-service/web_server.cpp.o

**Comment by wassfila** [comment]
Hi, if you want, I can assist you in debugging your issue, and provide hints how to narrow it down.

- Could you provide the steps you followed, one by one, or post the link of the exact tutorial you followed ?
- post an attached file with the full error log including all commands you typed.
- Did you try instal on a fresh raspberry pi install or did you try on one you ised for something else before, some already installed programs could conflict with the install.
- what is your end goal ? What is your hw and sensors set up? Maybe there could be an easier way to start.
- Also, depending on if you're a student, a hobbyist or a startup, I can suggest different approaches, depening on the expected learning curve.
